components/_expanding-components/uncheck-radio-button.js

```diff
@@ -0,0 +1,37 @@
+export const UncheckRadioButton = function(elementClass) {
+  const expandingComponent = elementClass
+  let expandingElement
+
+  const cacheDom = function() {
+    expandingElement = document.querySelectorAll('.' + expandingComponent)
+  }
+
+  const bindEvents = function() {
+    expandingElement.forEach(labelElement => {
+      labelElement.addEventListener('click', () => findInputForLabel(labelElement))
+    })
+  }
+
+  const findInputForLabel = function(labelElement) {
+    const labelElementFor = labelElement.htmlFor
+    const inputId = labelElementFor
+
+    if (labelElementFor) {
+      uncheckRadioButton(inputId)
+    }
+  }
+
+  const uncheckRadioButton = function(inputId) {
+    const inputElement = document.getElementById(inputId)
```

Comment:
this is cached from dom and should not be in this method.

Reply:
good call, but I can't figure out a better name, do you have any suggestions? The method unchecks a radio button...

Reply:
there is no way that I can pass the input id in the cached dom.... the only place I can pass the id is here